src/main/User/Services/DeleteUserService.java

```diff
@@ -0,0 +1,118 @@
+package User.Services;
+
+import Config.Message;
+import Config.Service;
+import Database.User.UserDao;
+import PDF.PDFType;
+import Security.SecurityUtils;
+import User.User;
+import User.UserMessage;
+import Validation.ValidationUtils;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.gridfs.GridFSBucket;
+import com.mongodb.client.gridfs.GridFSBuckets;
+import com.mongodb.client.gridfs.model.GridFSFile;
+import com.mongodb.client.model.Filters;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+public class DeleteUserService implements Service {
+  private MongoDatabase db;
+  private UserDao userDao;
+  private String username;
+  private String password;
+  private User user;
+
+  public DeleteUserService(MongoDatabase db, UserDao userDao, String username, String password) {
```

Comment:
I think ultimately, we don't want to be passing the MongoDatabase here. In deleting Documents, we should rely on the current instance of the PdfDao, and we can use the getAll() or delete() method in the dao object to do so.